src/webgl/contourFeature.frag

```diff
@@ -25,7 +25,7 @@ void main () {
         step = steps - 0.5;
       }
     } else {
-      step = valueVar;
+      step = valueVar + 0.5;
```

Comment:
what is 0.5 added to the valueVar?  I did not look at the complete code

Reply:
is valueVar range values or values?

Reply:
I should have documented this better as a bug fix and not directly related to the rest of the PR.  Specifically, we have a small texture which is the color map of the colors for the normalized range values.  The color of the fragment is interpolated from the texture.  Without the `+ 0.5`, it was picking the wrong color: with `+ 0.5` an integer value will be centered in texture's pixel coordinate and therefore that exact color.  In our examples, this is a relatively subtle shift.  You can construct an example where it is more obvious (for instance, a two color range where a lot of the values are at the high and low values would have shown some areas as halfway between the two colors instead of exactly at one of them).

Reply:
Okay thanks, that clarifies